TCAT.xcodeproj/project.pbxproj

```diff
@@ -837,7 +841,7 @@
 			baseConfigurationReference = 81725F2EBEBD3D184BFD0635 /* Pods-TCAT.release.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				DEVELOPMENT_TEAM = FV2G57GLFW;
```

Comment:
This has messed me up before, try to leave it the same so we don't get errors when we use master.